tests/config_test.py

```
@@ -308,10 +309,12 @@ def test_load_config_invalid_toml(self):
             f.flush()
 
             try:
-                with pytest.raises(
-                    Exception, match="[Ee]xpected"
-                ):  # Should raise a TOML parsing error
+                with pytest.raises(ConfigError, match="[Ee]xpected") as excinfo:
```

Ruff (0.16.3)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)